### DISCUSSION TITLE
Just getting started on Vim  installed LazyVim  run a Checkhealth

### DISCUSSION THREAD

**Discussion by thorrrr** [LazyVim/LazyVim#2083] [Q&A]
Please can somebody tell me if this Checkhealth report looks ok!
See [here](https://xbackbone.osopolar.cloud/VIDE3/RujUhAye08.) 
Or is there some things need fixing?

**Comment by dpetka2001** [comment]
The most noticeable to me seems to be that it cannot find `npm` in your PATH (installed with NodeJs) and also you seem to not have `python-pynvim` and `pip` for your Python version installed. Rest seem pretty ok and time will tell if you need anything else.

**Reply by thorrrr** [reply]
HI 
I am using kickstart.nvim so not sure if that is why. But if i try to install python-pynvim and pip in Arch via terminal it says target not found!

**Reply by dpetka2001** [reply]
For Arch you can see in Neovim [repo](https://github.com/neovim/neovim/wiki/Installing-Neovim#arch-linux) how to install `pynvim` and as for `pip` it seems that the package is `python-pip`. Unfortunately, I don't use Arch so I can't help any further than my google search allows me. You should google for any further information you might need additionally.

**Reply by thorrrr** [reply]
HI 
Cheers i have installed them now thank you for help it looks better i think 
https://xbackbone.osopolar.cloud/VIDE3/YoSIWOmO51.

**Reply by dpetka2001** [reply]
Are you able to install packages via `mason`? A lot of packages use `npm` to install and from your log it seems that it still can't find `npm` in your PATH.

**Reply by thorrrr** [reply]
Hopefully its better now 

https://xbackbone.osopolar.cloud/VIDE3/nOVuZUKA52.

**Reply by dpetka2001** [reply]
Should be good. Most of the rest of warnings really depend on your usage of Neovim.
